src/vos/tests/evt_ctl.c

```diff
@@ -939,7 +946,9 @@ test_evt_iter_flags(void **state)
 	rc = evt_create(EVT_FEAT_DEFAULT, 13, arg->ta_uma, arg->ta_root,
 			DAOS_HDL_INVAL, &toh);
 	assert_int_equal(rc, 0);
-	data = (int **)malloc((NUM_EPOCHS+1)*sizeof(int *));
+	D_ALLOC_ARRAY(data, ((NUM_EPOCHS+1)*sizeof(int *)));
+	if (data == NULL)
+		goto end;
 	for (count = 0; count < NUM_EPOCHS+1; count++) {
 		data[count] = (int *)calloc(
```

Comment:
Should use D_ALLOC_ARRAY here too.   Again, it only needs the number of entries as it will do sizeof(*ptr) for you.   It's also less error prone since the type of data can be changed without changing the sizeof(int).

Reply:
Done